backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/humanaethica/participation/ParticipationService.java

```diff
@@ -69,6 +81,14 @@ public ParticipationDto createParticipation(Integer activityId, ParticipationDto
         if (activityId == null) throw  new HEException(ACTIVITY_NOT_FOUND);
         Activity activity = activityRepository.findById(activityId).orElseThrow(() -> new HEException(ACTIVITY_NOT_FOUND, activityId));
 
+        if (userid == participationDto.getVolunteerId()){
```

Comment:
This should be in the access control, only a member of the institution can create a participation